php/event_manage.php

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link href="http://localhost/evem/css/index.css" type="text/css" rel="stylesheet"/>
```

Comment:
```suggestion
     <link href="css/index.css" type="text/css" rel="stylesheet"/>
```
Change this in relative to root folder evem